Voting Body: Add missing params (#106)
near-ndc/voting-v1: voting_body/README.md

Syntax: #vote_type denotes number of votes of the specified type, eg: #approve means number of approve votes.

A proposal is **approved** when:

- voting time is over
- AND consent is reached (quorum + threshold).

A proposal is marked as **spam** when:

- `#spam > #reject`
- AND `#reject + #spam >= (1-threshold) * (#approve + #reject + #spam)`

Spam proposals are removed, and the bond is slashed (sent to the community treasury).

> A proposal is **rejected** if voting time is over (proposal is not in progress any more), and it was not approved nor marked as spam.

Reviewer: ```suggestion
A proposal is **rejected** if voting time is over (proposal is not in progress anymore), and it was not approved nor marked as spam.
```
